include/transaction/esdt.h

```diff
@@ -4,15 +4,43 @@
 #include "transaction.h"
 #include "scarguments.h"
 
-#define NO_FUNCTION std::string()
-#define NO_PARAMETERS SCArguments()
-
-#define ESDT_GAS_LIMIT_NO_FUNCTION 500000
 #define ESDT_TRANSFER_PREFIX std::string("ESDTTransfer")
+#define ESDT_TRANSFER_GAS_LIMIT_NO_FUNCTION 500000
+#define ESDT_TRANSFER_NO_FUNCTION std::string()
+#define ESDT_TRANSFER_NO_ARGUMENTS SCArguments()
+
+#define ESDT_ISSUANCE_PREFIX std::string("issue")
+#define ESDT_ISSUANCE_VALUE "50000000000000000" //(0.05 EGLD)
+#define ESDT_ISSUANCE_GAS_LIMIT 60000000
```

Comment:
https://github.com/ElrondNetwork/elrond-config-testnet/blob/master/gasSchedules/gasScheduleV3.toml#L28

Perhaps rewrite the constant as 50000000 + 10000000, or perhaps compute the exact gas limit after building the transaction data?

Reply:
Based on docs :https://docs.elrond.com/developers/esdt-tokens/#issuance-of-fungible-esdt-tokens it just states that it should be 60000000, but in the link you attached it seems to be just 50000000. That seems odd.

> perhaps compute the exact gas limit after building the transaction data?

I think this is the right way to go. We should add function to compute transaction gas limit in the future.